resolwe/test/testcases.py

```diff
@@ -281,7 +281,7 @@ def setUp(self):
         self._register_schemas()
 
         self.collection = Collection.objects.create(contributor=self.admin, name="Test collection")
-        self.files_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'files')
+        self.files_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'toolkit', 'tests', 'files'))
```

Comment:
But at the moment test files are not used anywhere (if directory doesn't even exist till now). I think that right solution, if needed, would be to include general test processes in `test` app and not use real processes from other apps.

Or maybe even better would be to find a way to find the path to where the code is executed and not where it lives.